cirq-core/cirq/ops/common_channels.py

```diff
@@ -779,6 +780,10 @@ def reset(qubit: 'cirq.Qid') -> raw_types.Operation:
     return ResetChannel(qubit.dimension).on(qubit)
 
 
+R = reset
+document(R, """Alias for cirq.reset""")
```

Comment:
This overwrites the reset docstring [here](https://github.com/quantumlib/Cirq/blob/24a0d2e60caa25bad135ea8b737beb308e4fad57/cirq-core/cirq/_doc.py#L44).

Perhaps we should just add a note about the `R` alias to the reset docstring?

Reply:
If we want references to cirq.M in the generated docs, we might need to create a wrapper function so it is a different object.